.github/workflows/testing.yml

```diff
@@ -88,8 +88,16 @@ jobs:
     - name: Install nox
       run: python -m pip install --progress-bar off --upgrade nox
 
-    - name: Install graphviz and pandoc
-      run: sudo apt install graphviz pandoc
+    - name: Install graphviz
+      run: sudo apt install graphviz
+
+    - name: Install pandoc
+      run: |
+        PANDOC_VERSION="3.1.11.1"
+        wget -q https://github.com/jgm/pandoc/releases/download/${PANDOC_VERSION}/pandoc-${PANDOC_VERSION}-1-amd64.deb
+        sudo dpkg -i pandoc-${PANDOC_VERSION}-1-amd64.deb
+        rm pandoc-${PANDOC_VERSION}-1-amd64.deb
+        pandoc --version
```

Comment:
This didn't fix the problem, but it did get rid of the warning.

Reply:
I will create an issue to ensure that we do not overlook this matter, as it could have a significant impact in the future.